lib/tools/adb-commands.js

```diff
@@ -231,7 +231,6 @@ methods.sendTelnetCommand = async function (command) {
         dataStream += data;
         if (readyRegex.test(data)) {
           res = dataStream.replace(readyRegex, "").trim();
-          res = _.last(res.trim().split('\n'));
```

Comment:
Without this the new Android studio does not work when auto launching the emulator. Is it causing problems?